tests/router_test.ts

```diff
@@ -1500,6 +1500,70 @@ scenarios.forEach(function(scenario) {
       });
   });
 
+  test('abort query param only', function(assert) {
+    assert.expect(6);
+    map(assert, function(match) {
+      match('/').to('index');
+    });
+
+    routes = {
+      search: createHandler('search'),
+    };
+
+    let newParam = false;
+    let initial = true;
+
+    router.updateURL = function(updateUrl) {
+      url = updateUrl;
+      if (!initial) {
+        assert.ok(false, 'updateURL should not be called');
```

Comment:
@rwjblue - I was looking for something like `currentURL()`, but being in router.js, couldn't really find it, so this was the best I could come up with. If you have other ideas I'm certainly open. This does fail without the fix and pass with it.